AppController/djinn.rb

```diff
@@ -4010,18 +4009,20 @@ def start_soap_server()
 
   def start_datastore_server
     db_master_ip = nil
+    db_proxy = nil
     verbose = @options['verbose'].downcase == 'true'
     @nodes.each { |node|
       db_master_ip = node.private_ip if node.is_db_master?
+      db_proxy = node.private_ip if node.is_shadow?
```

Comment:
Should we use load_balancer here? and account for possibly use more than one ip?